src/firmware/main/primitives/accurate_shoot.c

```diff
@@ -188,13 +187,28 @@ static void accurate_shoot_tick(log_record_t *log)
     }
     else
     {
+        FirmwareRobot_t* robot = app_firmware_world_getRobot(world);
+
         // accelerate at ball to kick it
-        chicker_auto_arm((global_params->extra & 1) ? CHICKER_CHIP : CHICKER_KICK,
-                         global_params->params[3]);
-        if (!(global_params->extra & 1))
+        Chicker_t* chicker = app_firmware_robot_getChicker(robot);
+        bool chipping      = global_params->extra & 1;
+        if (chipping)
+        {
+            float chip_distance = global_params->params[3];
```

Comment:
Nit:
Would be nice to have a constant here for the '3'

Reply:
Agreed, deferring this until we migrate to protobuf (and so need to change this anyhow)